R/tree-map-alignment.R

```diff
@@ -593,10 +926,17 @@ vis1 = function(trees) {
 # on the observed trees, with some buffer. Assumes the observed tree map is 50 x 50 and that both
 # tree maps are centered at 0,0.
 #' @export
-vis2 = function(pred, obs, obs_foc = FALSE, coords_arbitrary = FALSE, zoom_to_obs = FALSE, obs_buffer = 75) {
+vis2 = function(pred, obs, shift=c(0, 0), obs_foc = FALSE, coords_arbitrary = FALSE, zoom_to_obs = FALSE, obs_buffer = 75) {
   pred = pred |> dplyr::mutate(layer = "predicted")
   obs = obs |> dplyr::mutate(layer = "observed")
 
+  if (! identical(shift, c(0, 0))){
+    print(shift)
+    pred = pred |> dplyr::mutate(x = x + shift[1])
+    pred = pred |> dplyr::mutate(y = y + shift[2])
```

Comment:
This could be streamlined slightly:
```
    pred = pred |> dplyr::mutate(x = x + shift[1],
                                                  y = y + shift[2])
```